make/source.mk

```diff
@@ -200,6 +200,8 @@ COMMON_SRC = \
             io/vtx_smartaudio.c \
             io/vtx_tramp.c \
             io/vtx_control.c \
+            io/vtx_msp.c \
+            cms/cms_menu_vtx_msp.c \
```

Comment:
Additional `\` needs one more line to be empty. Jus to let you decide if we want that.

Reply:
removed